src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs

```diff
@@ -8,7 +8,7 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// <summary>
 /// <see cref="UIView"/> for <see cref="Alert"/>
 /// </summary>
-public class AlertView : UIView
+public sealed class AlertView : UIView
 {
 	readonly List<UIView> children = Array.Empty<UIView>().ToList();
```

Comment:
```suggestion
	readonly List<UIView> children = Enumerable.Empty<UIView>().ToList();
```

This will be faster

```
|               Method |     Mean |    Error |   StdDev | Ratio |  Gen 0 | Allocated |
|--------------------- |---------:|---------:|---------:|------:|-------:|----------:|
|       UsingArrayEmpty | 17.36 ns | 0.164 ns | 0.145 ns |  1.00 | 0.0051 |      32 B |
| UsingEnumerableEmpty | 10.22 ns | 0.060 ns | 0.047 ns |  0.59 | 0.0051 |      32 B |
```